client/my-sites/checkout/checkout-thank-you/jetpack-checkout-siteless-thank-you.tsx

```diff
@@ -41,12 +41,14 @@ interface Props {
 	forScheduling: boolean;
 	productSlug: string | 'no_product';
 	receiptId?: number;
+	source?: string;
 }
 
 const JetpackCheckoutSitelessThankYou: FC< Props > = ( {
 	forScheduling,
 	productSlug,
 	receiptId = 0,
+	source = 'onboarding-calypso-ui',
```

Comment:
This is the default value. For now, the other option is `onboarding-email` which comes from the link present in the onboarding email.